<launch>

    <arg name="imu_topic_name" default="/imu/raw_data"/>
    <arg name="point_num" default="200"/>
    <arg name="gravity" default="9.81"/>

    <node pkg="common" type="imu_calibration" name="imu_calibration" output="screen">
        <param name="imu_topic_name" type="string" value="$(arg imu_topic_name)"/>
        <param name="point_num" value="$(arg point_num)"/>
        <param name="gravity"   value="$(arg gravity)"/>
    </node>

</launch>